server/api/team/team.spec.js

```diff
@@ -62,3 +66,82 @@ describe('GET /api/teams/:id/players', function() {
       });
   });
 });
+
+
+describe ('POST /api/teams/fill', function() {
+  var account1 = {
+    email: 'test1@test.com',
+    password: 'test',
+    role: 'admin'
+  };
+
+  var account2 = {
+    email: 'test2@test.com',
+    password: 'testing',
+    role: 'commish'
+  };
+
+  before(function(done) {
+    // Clear db before testing
+    db.Sport.destroy({}, {truncate: true}).then(function() {
+      return db.User.destroy({}, {truncate: true})
+    }).then(function() {
+      return db.Team.destroy({}, {truncate: true})
+    }).then(function() {
+      return db.Player.destroy({}, {truncate: true})
+    }).then(function() {
+      return db.PlayerAssignment.destroy({}, {truncate: true})
+    }).then(function() {
+      done();
+    });
+  });
+
+  before(function(done) {
+    db.User.create(account1).then(function(adminUser) {
+      return fillTest.fillTeams(adminUser);
+    }).then(function() {
+     done();
+    });
+  });
+
+  it('should have prepared 5 teams', function(done) {
+    db.Team.findAndCountAll().then(function(result) {
+      return result.count.should.equal(5);
+    }).then(function() {
+      done();
```

Comment:
Wouldn't it mean that it'd hit done the minute it finishes reading the first line though? I suppose in Mocha if that fails then it'd auto-fail the test, though.